class/defaults.yml

```
@@ -10,6 +10,6 @@ parameters:
     provider: ${facts:cloud}
     version: master
     terraform_variables:
-      source: github.com/appuio/terraform-openshift4-${openshift4_terraform:provider}?ref=${openshift4_terraform:version}
+      source: git::https://github.com/appuio/terraform-openshift4-${openshift4_terraform:provider}.git//?ref=${openshift4_terraform:version}
```

in the `tests/*.yaml` the URL is appended without `.git`. Intended? I guess they should be consistent
```suggestion
      source: git::https://github.com/appuio/terraform-openshift4-${openshift4_terraform:provider}//?ref=${openshift4_terraform:version}
```

---

There's actually no reason to have the URL explicitly in the test input, I've cleaned up the tests a bit.

---

ah, that's better